### PR TITLE
DEVELOPER-3578 Updated CSS for alerts

### PR DESCRIPTION
**Pull request by adelasofia**
[DEVELOPER-3578](https://issues.jboss.org/browse/DEVELOPER-3578) - Update alert styling: CSS classes.

Update the alert styles used on the website to be consistent with the recommendations in the Red Hat Unified Style Guide. This likely includes CSS changes as well as changes to the location of the alert boxes on the page.

DO NOT MERGE YET, APPROVAL PENDING

### CODE REVIEW

**Comment by robpblake** [comment]
retest this please

**Comment by KamiQuasi** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please

**Comment by KamiQuasi** [comment]
retest this please

File: stylesheets/_alerts.scss [hunk 1]
```diff
@@ -1,49 +1,98 @@
...
+  }
 
   &.alert-info {
     background-color: #dcedf8;
     border-color: #87aac1;
 
-    &.icon {
-      background-image: cdn("../images/icons/alerts/RHD_alerticon_info.png");
+    div.icon {
+      &:before { content: url('http://static.jboss.org/rhd/images/icons/RHD_alerticon_info.svg'); }
```
Comment (lkrzyzanek) [inline stylesheets/_alerts.scss]: @adelasofia would be better to use either https or just use //static... to avoid "mixed content" warning

Comment (adelasofia) [reply stylesheets/_alerts.scss]: Sorry about that, I didn't realize I was using http. I'm raising a PR to fix this 😄

**Pull request merged**